migrations/less_items_#291.py

```diff
@@ -0,0 +1,23 @@
+import os
+import sys
+sys.path.insert(0, os.path.abspath(os.path.join(sys.argv[0], '..', '..')))
```

Comment:
Oh god. 
This type of scripts we need to call from root of the project so, there is no need to add something to path